crates/parser/src/lexed_str.rs

```diff
@@ -187,6 +187,12 @@ impl<'a> Converter<'a> {
                 }
 
                 rustc_lexer::TokenKind::RawIdent => IDENT,
+
+                rustc_lexer::TokenKind::GuardedStrPrefix => {
```

Comment:
Yuck we need to break the token apart here in < Edition2024

Reply:
Yeah, I'm afraid I'm gonna let you handle that :sweat_smile:.

Reply:
Will see when I get to that. I was also confused as to how this didn't break our test upstream but I forgot upstream doesn't run our tests (except for proc-macro-srv) ... We should make them run them :^)

Reply:
I thought they were disabled because of the impact on CI time, but it was probably being built with optimizations: https://github.com/rust-lang/rust/pull/113244.